task/03-date-tasks.js

```diff
@@ -56,8 +56,7 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
-}
+   return date.getFullYear() % 4 != 0? false: date.getFullYear() % 100 != 0? true:date.getFullYear() % 400 == 0? true: false}
 
```

Comment:
not a clear code, do not use a “question mark” operator when there're many conditions
separate the steps in the code